properties.cpp

```diff
@@ -28,15 +28,31 @@ void Properties::DisplayProperties(QVariant p) {
 
   // only support QVariantMap or QVariantHash at this level
```

Comment:
This comment is now wrong

Reply:
Yes and no.
The third option is a list which holds implicitly QVariantMap or QVariantHash.

But I did not completely figured out what is supported here also.
From what I have learned through debugging, only the first and third options are used in current code.